AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -123,6 +123,9 @@ ayedas.com.tr,toroslaredas.com.tr,baskentedas.com.tr##.footer_cerezpolitikasi
 enerjisa.com.tr##.enerjisa-cerez-politikasi-cookie-bar
 xoxowifi.com###cookies_message_container
 tylko.com##.ty-settings
+momondo.com#$#div[id$="data-transfer-disclaimer-dialog"] { display: none !important; }
+momondo.com#$#.keel-mom.modal-open { position: static !important; }
+momondo.com#$#.hideOverflow { overflow: auto !important; }
```

Comment:
It's better not to miss `body` or `html` tag when you change their overflow value.
```
momondo.com#$#body.hideOverflow { overflow: auto !important; }
```

Reply:
Ah wait, these two rules about `body`. You can improve them
```
momondo.com#$#body.hideOverflow { position: static !important; overflow: auto !important; }
```

Reply:
Btw, do you need them for other languages? I see that only on `cn.momondo.com`.